site/app/templates/error/genericError.twig

```diff
@@ -0,0 +1,5 @@
+<div class="content">
+    {% for msg in error_messages %}
```

Comment:
I think we should just make this page show a singular error message until such a time as we actually need to show multiple error messages, as I'm not sure we'll ever hit that point.

Reply:
I'm trying to promote flexibility and re-usability.  You'd prefer I change to a less flexible, less re-usable design?  You don't have to pass discrete error messages, one could instead pass multiple sentences they want formatted onto separate lines instead of one single paragraph.

Reply:
And my argument is that I'm not necessarily sure we'll ever use that feature, so why have it. Designing things for maximum flexibility and reusability "just in case" is as much an anti-pattern as not having any flexibility / reusability, if that "just in case" never happens.

But this is not a hill I will die on.